integrated_channels/blackboard/client.py

```diff
@@ -37,7 +41,34 @@ def __init__(self, enterprise_configuration):
         self.expires_at = None
 
     def create_content_metadata(self, serialized_data):
-        """TODO"""
+        """
+        Create a course from serialized course metadata
+        """
+        if not serialized_data:
+            return 204, "No courses created, empty content_metadata_items"
```

Comment:
Wouldn't this normally be a noop anyways? IE you should never even call this method if you don't have data?

Reply:
yeah I think we can scrap this one and assume we will get valid data otherwise we will fail later anyway